tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -0,0 +1,448 @@
+import os
+import pytest
+from .common import *  # NOQA
+from .test_bkp_restore import wait_for_backup_to_active
+from .test_rbac import create_user
+from rancher import ApiError, Client
+
+DO_ACCESSKEY = os.environ.get('DO_ACCESSKEY', "None")
+RANCHER_S3_BUCKETNAME = os.environ.get('RANCHER_S3_BUCKETNAME', "None")
+RANCHER_S3_ENDPOINT = os.environ.get('RANCHER_S3_ENDPOINT', "None")
+AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', "None")
+AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY', "None")
+RANCHER_DEFAULT_K8S_VERSION = os.environ.get('RANCHER_DEFAULT_K8S_VERSION',
+                                             "v1.14.8-rancher1-1")
+
```

Comment:
v3/settings/k8s-versions-current should have the list of supported K8s versions . This variable should take values from here.

Reply:
Done